src/IDCodeValidator.java

```diff
@@ -0,0 +1,193 @@
+import java.util.*;
+
+public class IDCodeValidator {
+
+
+    public static boolean isCorrect(String idCode) {
+        System.out.println(idCode);
+        IDNumberData personalCode = IDNumberData.getInformationFromIdCode(idCode);
+        if (idCode.length() == 11
+                && isGenderNumberCorrect(personalCode.genderNumber)
+                && isDayNumberCorrect(personalCode.dayNumber)
+                && isMonthNumberCorrect(personalCode.monthNumber)
+                && isBirthDateCorrect(personalCode.yearNumber, personalCode.monthNumber, personalCode.dayNumber)
```

Comment:
Here you would need to pass the full year, instead of just yearNumber as value of yearNumber is between 0 and 99. You can use your `getFullYear` method here